torch/distributed/_tensor/_op_schema.py

```diff
@@ -271,24 +271,24 @@ def __repr__(self) -> str:
         )
 
     def __str__(self) -> str:
-        args_sharding: List[str] = []
+        args_schema: List[str] = []
```

Comment:
The reason why we don't use `args_schema` is that it is confliciting with `OpSchema.args_schema`, so we want to distinguish that, why? `args_schema` could consists of non Tensor/DTensor arguments, while args_sharding means we only want to include DTensor sharding into the list, excluding non-Tensor/DTensor args

Reply:
Follow up question: I thought everything else also get append to args_sharding from here:
https://github.com/pytorch/pytorch/blob/main/torch/distributed/_tensor/_op_schema.py#L289-L290

Reply:
hmm actually you are right, my bad